testing/environments/docker/kibana/Dockerfile-snapshot

```diff
@@ -1,19 +1,16 @@
 # Copy of https://github.com/elastic/kibana-docker/blob/master/build/kibana/Dockerfile
 FROM centos:7
 LABEL maintainer "Elastic Docker Team <docker@elastic.co>"
-
 EXPOSE 5601
 
+# Beats specific args
```

Comment:
@jarpy Below is one the main difference between the original image (https://github.com/elastic/kibana-docker/blob/master/templates/Dockerfile.j2) and what we use for beats. The reason we use `ARG` as this allows us to use one Dockerfile and one docker-compose file an only change the args depending on the environment. Any idea how we could rely even more on the official images?